data/hai/hai reveal 4 middle.txt

```diff
@@ -0,0 +1,455 @@
+			`	"While the crime in Hai space is rather more manageable, the same lack of evidence makes it hard for the Elders to justify a dedicated task force when more visible priorities still dominate the news feeds. They tell me that if we can prove the kidnappings are from a coordinated effort, they'll put all the resources they can spare into stopping the crime. So far, they say, their investigators have been unable to discover anything."`
+			`	"What we need is a skilled pilot with a jump drive to visit some locations we cannot easily access."`
+				goto questions
+			label help precious opinions  
+			`	"We have some leads in human and Korath space we need you to follow. The Quarg at Alta Hai had me talk to scientists on <destination>. They say they can help, but they need a sample of the microtransceivers. I know the humans gave you a Jump Drive; I need you to use it to deliver the microtransceivers."`
+			`	She hands you a data card. "This has some info on it that you might find useful. Right now though, the Quarg in Korath space might eliminate some leads and add new ones. Follow the leads. Once you have proof the kidnappings are an organized effort, return here, and we will pay you 2,000,000 credits."`
+			choice
+				`	"I'll start right away."`
+					accept
+				`	"No, I'm not interested."`
+					decline
+
+# Must be after the Quarg and Korath intro missions:
+mission "ZZ Hai Rescue [A03] Microtransceiver Identification"
+	landing
+	name "Missing Researchers"
+	description "Fly to <destination> to talk to Tarazed Shipyards about the microtransceivers."
+	source "Kuwaru Efreti"
+	destination "Wayfarer"
+	to offer
+		has "Hai Rescue [A02] Microtransceiver Identification: done"
+	on offer
+		log `The Korath identified the microtransceivers as using the same materials and design principles as their own technology, but extremely primitive and poorly-made. The Quarg provided a data card that Tarazed Shipyards may be able to use.`
+		conversation
+			`After you land, a hovering gray platform lowers to the entrance of your ship. You step on, and it raises you to a higher level of the ringworld where many Quarg and a few Korath are present. You find the first Quarg who will talk to you and show it the vial of microtransceivers.`
+			`	The Quarg makes a series of wheezing sounds which approximate a laugh. "I am merely a pilot. I know nothing of microtransceivers. That is the technology of the Korath, and I will take you to one who is knowledgeable."`
+			`	You pass through another two grey platform transports and reach a warm cave-like structure with a group of Korath and Quarg. From the holograms and equipment, it appears they're discussing science. One hologram looks like a blood cells attached to a tiny machine; this may be a microscale medicine laboratory.`
```

Comment:
```suggestion
			`	You pass through another two gray platform transports and reach a warm cave-like structure with a group of Korath and Quarg. From the holograms and equipment, it appears they're discussing science. One hologram looks like a blood cells attached to a tiny machine; this may be a microscale medicine laboratory.`
```